src/pages/basic_data/institution_admin/person_register/PersonRegisterPage.tsx

```diff
@@ -98,7 +87,7 @@ export const PersonRegisterPage = () => {
                 </TableRow>
               </TableHead>
               <TableBody>
-                {isLoadingEmployees
+                {employeeSearchQuery.isLoading && !employeeSearchQuery.data
```

Comment:
Trengs ikke når vi har keepPreviousData
```suggestion
                {employeeSearchQuery.isLoading
```